guides/05_custom-components/07_pdf-component-example.md

```diff
@@ -0,0 +1,672 @@
+# Case Study: A Component to Display PDFs
+
+Let's work through an example of building a custom gradio component for displaying PDF files.
+This component will come in handy for showcasing [document question answering](https://huggingface.co/models?pipeline_tag=document-question-answering&sort=trending) models, which typically work on PDF input.
+
+## Step 0: Prerequisites
+Make sure you have gradio 4.0 installed.
+As of the time of publication, the latest release is 4.1.1.
```

Comment:
Gets outdated, I don't think we need to mention the specific version
```suggestion
Make sure you have gradio 4.0 or higher installed (run `pip install --upgrade gradio`).
```

Reply:
Its likely we're going to make improvements to the CLI experience or some internal APIs so I want to mention the version so that someone can follow this guide exactly.

Reply:
Got it. We can upgrade to 4.2.0